sycl/include/sycl/esimd/detail/esimd_types.hpp

```diff
@@ -0,0 +1,249 @@
+//==-------------- esimd_types.hpp - DPC++ Explicit SIMD API ---------------==//
```

Comment:
duplicate of https://github.com/intel/llvm/pull/1853/files#diff-a6b3420bb6f733913e3f1d168adadf32

Reply:
Please review only 3rd commit:
[SYCL][ESIMD] Explicit SIMD math APIs.